README.md

```diff
@@ -155,7 +155,4 @@ Additional tooling can be found in [/docs/tools](/docs/tools).
 
 ### Research
 
-- [The latest gossip on BFT consensus](https://arxiv.org/abs/1807.04938)
-- [Master's Thesis on Tendermint](https://atrium.lib.uoguelph.ca/xmlui/handle/10214/9769)
-- [Original Whitepaper: "Tendermint: Consensus Without Mining"](https://tendermint.com/static/docs/tendermint.pdf)
-- [Blog](https://blog.cosmos.network/tendermint/home)
+We are hiring Go engineers! Join us in building the future of blockchain scaling and interoperability. [Apply here](https://jobs.lever.co/celestia).
```

Comment:
This does not look right btw: https://github.com/celestiaorg/celestia-core/blob/e499d0f5c6bef18ba360eb0ea6c098e7a8d2a6b6/README.md